README.md

```diff
@@ -18,7 +18,20 @@ standard.
 Translations of the information model may be available.
 English remains the normative language in all cases.
 
-## Branch Structure
+[![Core Profile diagram](./images/model-Core.png "Core Profile diagram")](./images/model-Core.png)
```

Comment:
Do we want to have the diagram here?  Seems like we'd have to scroll more to get to the other content.

Reply:
Agree that it's too much of the scroll. Too larger than I was expected.
I have removed the full size diagram from here and instead add a 120-pixel-height version under the Model heading. What do you think?